testing/file_test/file_test_base.cpp

```diff
@@ -147,9 +156,11 @@ auto FileTestBase::ProcessTestFileAndRun(TestContext& context)
 
   // Pass arguments as StringRef.
```

Comment:
This is no longer accurate, can you please update it?

Reply:
Done.